main.nf

```diff
@@ -1539,86 +1540,87 @@ process samtools_filter {
     tuple samplename, libraryid, lane, seqtype, organism, strandedness, udg, file("*.unmapped.bam") optional true
 
     // Using shell block rather than script because we are playing with awk
```

Comment:
```suggestion

```
This will become confusing in the future if we are no longer using a shell block